plat/imx/common/include/imx_sip_svc.h

```diff
@@ -18,6 +18,9 @@
 #define IMX_SIP_BUILDINFO_GET_COMMITHASH	0x00
 
 #define IMX_SIP_SRC			0xC2000005
+#define IMX_SIP_SRC_M4_START		0x00
+#define IMX_SIP_SRC_M4_STARTED		0x01
+#define IMX_SIP_SRC_M4_STOP		0x02
```

Comment:
Is this aligned with _U-Boot_? From the commit message, it is not clear.

Reply:
Found the info myself: this is already present in the _U-Boot_ tree in `include/imx_sip.h`. This was introduced with commit [`de274663f542 ("imx: add sip function")`](https://github.com/hexagon-geo-surv/u-boot/commit/de274663f5429fe10c48a40f9fbf0c8aa553905d).

`IMX_SIP_SRC_M4_STOP` came with commit [`c383379f25e2 ("imx: imx9: Support booting m33 from Acore")`](https://github.com/hexagon-geo-surv/u-boot/commit/c383379f25e2491d9502f367324744cc580c0eb2)